stetho/src/main/java/com/facebook/stetho/inspector/elements/Descriptor.java

```diff
@@ -2,24 +2,39 @@
 
 package com.facebook.stetho.inspector.elements;
 
+import com.facebook.stetho.common.Util;
+
 import javax.annotation.Nullable;
 
 public abstract class Descriptor implements NodeDescriptor {
-
-  private Listener mListener;
+  private Host mHost;
 
   protected Descriptor() {
   }
 
-  void setListener(Listener listener) {
-    mListener = listener;
+  // This is called by DescriptorMap to hook us up to our Host (probably the DOMProvider).
```

Comment:
nit: use Javadocs and `{@link}`.  Because this is package private, it is safe to document it in this way.